cmd/version.go

```diff
@@ -27,12 +30,11 @@ const unknown = "unknown"
 // information in the release process
 var (
 	kubeBuilderVersion      = unknown
-	kubernetesVendorVersion = unknown
+	kubernetesVendorVersion = "1.31.0"
```

Comment:
I took the liberty of making this the default value originally from https://github.com/kubernetes-sigs/kubebuilder/blob/master/build/.goreleaser.yml. Potentially, this value could be tracked solely here to avoid having to update 2 places every time.

If it is preferred to keep the value as `unknown`, let me know.

Reply:
I think it is fine, but we need to bump 1.32.0

Reply:
https://github.com/kubernetes-sigs/kubebuilder/pull/4517